bin/Powheg/production/pre2017/13TeV/gg_H_quark-mass-effects_JHUGenV698_HWWLNuQQ_NNPDF30_13TeV/JHUGen_gg_H_WWLNuQQ_reweightPMWW.input

```diff
@@ -0,0 +1 @@
+DecayMode1=10 DecayMode2=5 ReweightDecay WidthSchemeIn=3 ReadPMZZ
```

Comment:
I think you want to add
`PmHstarFile=PMWWdistribution.out`
Otherwise you're reading the distribution for HZZ decay, which is wrong.

Reply:
Inside the gridpack we have the PMZZdistribution.out file but, despite the name, it was correctly generated for WW. This was discussed in an old thread with the JHUGen authors.
If you want to check the gridpacks you can find them here:

    /afs/cern.ch/user/d/dmoran/public/HiggsGridPacks

Reply:
Ah, got it!  I remember there were several iterations on the procedure (I was the one who developed it in both JHUGen and here).

If you create new gridpacks using run_pwg, the PMWWdistribution.out is automatically downloaded, so then you want the option as I said before.

I guess in principle the Pwhateverdistribution is also an "input card" and should probably be documented.

Reply:
These gridpacks have not been created from scratch. I have used the existing fully leptonic gridpacks and just changed the JHUGen input card.

I will add the PMWWdistribution.out file as you suggest.

Reply:
sorry @lviliani didn't you just added the PMZZ files?

Reply:
The name is PMZZdistribution.out but the file actually contains the values for WW.
It was a naming issue related to this (old) version of JHUGen, that has been fixed in the newer version.
Sorry if I was not clear before.